packages/docs/src/routes/docs/(qwik)/components/tasks/index.mdx

```diff
@@ -160,10 +160,12 @@ export default component$(() => {
   });
 
   return (
-    <div>
-      Enter text: <input bind:value={text} />
-      <div>Delayed text: {delayText}</div>
-    </div>
+    <section>
+      <p>
+        Enter text: <input bind:value={text} />
```

Comment:
There's an opportunity to use `<label>` whenever there is an `<input>` to improve the semantics even more. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#examples

Reply:
Good catch @cmbartschat! I went back through and added <label> wrappers in a number of places where I thought they were semantically appropriate.